planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/src/scene_module/intersection/util.cpp

```diff
@@ -251,14 +253,15 @@ bool generateStopLine(
     // get idx of first_inside_lane point
     first_idx_ip_inside_lane = getFirstPointInsidePolygons(path_ip, detection_areas);
     if (first_idx_ip_inside_lane == -1) {
-      ROS_DEBUG("[Intersection Util] generate stopline, but no intersect line found.");
+      RCLCPP_DEBUG(logger, "[Intersection Util] generate stopline, but no intersect line found.");
       return false;
     }
     // only for visualization
     const auto first_inside_point = path_ip.points.at(first_idx_ip_inside_lane).point.pose;
     planning_utils::calcClosestIndex(*path, first_inside_point, *first_idx_inside_lane, 10.0);
     if (*first_idx_inside_lane == 0) {
-      ROS_DEBUG(
+      RCLCPP_DEBUG(
+        logger,
         "[Intersection Util] path[0] is already in the detection area. This happens if you have "
         "already "
```

Comment:
[nit] too many newlines, strings could be merged. Probably created by clang-format but not touch again later

Reply:
Done.